internal/store/builder.go

```diff
@@ -138,6 +146,23 @@ var availableStores = map[string]func(f *Builder) *metricsstore.MetricsStore{
 	"storageclasses":             func(b *Builder) *metricsstore.MetricsStore { return b.buildStorageClassStore() },
 }
 
+func collectorExists(name string) bool {
+	for s, _ := range availableStores {
+		if s == name {
+			return true
+		}
+	}
+	return false
+}
+
+func availableCollectors() []string {
+	c := []string{}
+	for name, _ := range availableStores {
```

Comment:
this is a map already, so we can just check on existance of the collector key in this map directly in `collectorExists`

Reply:
Fixed

Reply:
Are we still calling this new method `availableCollectors()`?